scrapers/Meninosonline.yml

```
@@ -0,0 +1,38 @@
+name: "MeninosOnline"
+sceneByURL:
+  - action: scrapeXPath
+    url:
+      - meninosonline.net
+    scraper: sceneScraper
+xPathScrapers:
+  sceneScraper:
+    scene:
+      Title: 
+        selector: //div[@class="col-sm-12"]/h3
+        postProcess:
+          - replace:
+              - regex: .* \(
+                with: ""
+              - regex: \) .*
+                with: ""
+      Date:
+        selector: //div[@class="col-sm-12"]/h3
+        postProcess:
+          - replace:
+              - regex: .* -.
+                with: ""
+          - parseDate: 2006-01-02
```

the regex used doesnt seem safe and the format used for parseDate is wrong
```suggestion
        postProcess:
          - replace:
              - regex: .*(\d{2}/\d{2}/\d{4})$
                with: $1
          - parseDate: 02/01/2006
```

---

Date format is language specific. I changed url to en site.